### PR TITLE
Filtration, Posts Service

### PR DESCRIPTION
**Pull request by JustRoxy**
And answer api route bugfix

### CODE REVIEW

File: backend/CircleForms/Models/Posts/Post.cs [hunk 1]
```diff
@@ -19,6 +19,9 @@ public Post()
     [JsonProperty("author_id")]
     public string AuthorId { get; set; }
 
+    [JsonProperty("is_active")]
+    public bool IsActive { get; set; }
```
Comment (stanriders) [inline backend/CircleForms/Models/Posts/Post.cs:23]: Might as well add `DateTime ActiveUntil` and related filtration checks (see #56)

File: backend/CircleForms/Models/Result.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    public Result(HttpStatusCode code, string message)
+    {
+        StatusCode = code;
+        Message = message;
+        IsError = true;
+    }
+
+    public Result(string message)
+    {
+        StatusCode = HttpStatusCode.BadRequest;
```
Comment (stanriders) [inline backend/CircleForms/Models/Result.cs:22]: `HttpStatusCode.InternalServerError`? I think if we know it's something related to requests being invalid, we'd know status code as well.

File: backend/CircleForms/Services/Database/PostRepository.cs [hunk 3]
```diff
@@ -156,6 +180,11 @@ public async Task AddAnswer(string postId, Answer entry)
         }
     }
 
+    private static long ToUnixTimestamp(DateTime time)
```
Comment (stanriders) [inline backend/CircleForms/Services/Database/PostRepository.cs:183]: Move to an extension class

**Review by stanriders — CHANGES_REQUESTED** [review]

File: backend/CircleForms/Models/Posts/Post.cs (hunk 1 above)
Comment (JustRoxy) [reply backend/CircleForms/Models/Posts/Post.cs:23]: later

File: backend/CircleForms/Models/Result.cs (hunk 2 above)
Comment (JustRoxy) [reply backend/CircleForms/Models/Result.cs:22]: This overload is specific to BadRequest messages, if you want to InternalServerError then throw an exception

File: backend/CircleForms/Services/Database/PostRepository.cs (hunk 3 above)
Comment (JustRoxy) [reply backend/CircleForms/Services/Database/PostRepository.cs:183]: resolve problems at the rate of occurring

Comment (stanriders) [reply backend/CircleForms/Services/Database/PostRepository.cs:183]: You're creating unnecessary tech debt

File: backend/CircleForms/Models/Result.cs (hunk 2 above)
Comment (stanriders) [reply backend/CircleForms/Models/Result.cs:22]: Then do it similarly to `public static Result<T> NotFound(string id)`? Not sure why we should have three different ways of creating a result

File: backend/CircleForms/Services/Database/PostRepository.cs (hunk 3 above)
Comment (JustRoxy) [reply backend/CircleForms/Services/Database/PostRepository.cs:183]: It's not a tech debt, it's a reduction of abstractions. Currently we don't need to use ToUnixTimestamp in other scenarios, so what's the point

**Review by stanriders — APPROVED** [review]

**Pull request merged**